### ISSUE TITLE
5.1.1版本设置了map，markPoint设置了symbolOffset无效

### ISSUE THREAD

**Issue by gabirellaq** [apache/echarts#15477]
### Version
5.1.1

### Steps to reproduce
- 示例代码：https://echarts.apache.org/examples/zh/editor.html?c=map-polygon
- series中name = 'Top5'处设置以下代码片段

```
markPoint: {
   symbol: 'circle',
   symbolSize: 80,
   symbolOffset: [-100, -100], // 无效
   data: [
       {name: "上海",coord: [121.48, 31.22]},
       {name: "合肥", coord: [117.27,31.86]},
       {name: "武汉", coord: [114.31,30.52]},
       {name: "大庆", coord: [125.03,46.58]}
   ],
},
```

### What is expected?
type是map的时候，设置mapPoint，symbolOffset生效

### What is actually happening?
type是map的时候，设置mapPoint，symbolOffset不生效

图片预览效果：https://user-images.githubusercontent.com/5259501/128137230-94fa33c1-dcda-4d07-af58-0f39b1188519.png

**Comment by liuyang77886** [comment]
我也有这个问题

**Comment by kchjxxgh** [comment]
known issue. I heard it would be fixed in 5.2.0

**Comment by Ovilia** [comment]
Fixed in #14737 which will be released in the coming 5.2.0.
